dataikuapi/dssclient.py

```diff
@@ -743,6 +743,19 @@ def log_custom_audit(self, custom_type, custom_params=None):
             body = custom_params)
 
     ########################################################
+    # Monitoring
+    ########################################################
+
+    def get_global_usage_summary(self):
+        """
+        Summarize the contents of the instance
+
+        :returns: a summary object
```

Comment:
Just to match the doc of other similar methods, I think we usually say it returns a dict rather than an object.